lacommunaute/notification/tests/tests_tasks.py

```diff
@@ -0,0 +1,45 @@
+import json
+from datetime import timedelta
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.utils import timezone
+
+from config.settings.base import DEFAULT_FROM_EMAIL
+from lacommunaute.forum_conversation.factories import PostFactory, TopicFactory
+from lacommunaute.notification.models import EmailSentTrack
+from lacommunaute.notification.tasks import send_notifs_when_first_reply
+
+
+class SendNotifsWhenFirstReplyTestCase(TestCase):
+    def test_send_notifs_when_first_reply(self):
+        topic = TopicFactory(with_post=True)
+        topic.first_post.created = timezone.now() - timedelta(hours=2)
+        topic.first_post.save()
+        post = PostFactory(topic=topic)
+        post.created = timezone.now() - timedelta(hours=1)
+        post.save()
```

Comment:
Pour s'éviter 1 requête ?
```suggestion
post = PostFactory(topic=topic, created=timezone.now() - timedelta(hours=1))
```